tests/interface/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -57,6 +60,10 @@ function testInterfaceName(o: I) {
   // error, name is static
   (o.constructor.name: string); // ok
 }
+
+declare module X {
+  declare interface Y { x: number }
```

Comment:
I've added this test to make sure that declare is added inside of a declare module block.